gcc-11.3.0/gcc/gcc.c

```diff
@@ -8047,6 +8741,13 @@ driver::main (int argc, char **argv)
     return get_exit_code ();
 
   do_spec_on_infiles ();
+  /* FIXME: Function create_symlinks_gitbom works only when compilation
+     step is being done (when -c option is being passed) and when the
+     name of the output file is specified with the -o option.  The support
+     for creating symlinks in the GitBOM concept when the GitBOM Document
+     file gitoid is extracted from the assembly file (when -S option is
+     used) should potentially be implemented as well.  */
+  create_symlinks_gitbom ();
```

Comment:
Could you enable the symlink creation only when environment variable  GITBOM_NO_EMBED is enabled ? This way, the patch will not interfere with the existing workflow (where we enable GITBOM_DIR).